apps/nowcasting-app/components/map/map.tsx

```diff
@@ -45,9 +47,16 @@ const Map = ({ loadDataOverlay, controlOverlay, bearing = 90 }: IMap) => {
       setZoom(map.current.getZoom().toFixed(2));
     });
 
-    map.current.on("load", () => {
+    map.current.on("load", (event) => {
       loadDataOverlay(map);
     });
+
+    // Indecated that data source updates every 5 minutes
+    // map.current.on("sourcedata", (event) => {
+    //   if(event.isSourceLoaded && event.sourceCacheId && event.source.data){
+    //     console.log('A data event occurred.', event);
+    //   }
+    // });
   });
```

Comment:
@Alaatohamy should be dependent  on map (maybe that is why it was loading source twice)
```suggestion
  },[map]);
```